fe/fe-core/src/main/java/com/starrocks/storagevolume/StorageVolume.java

```diff
@@ -82,6 +82,13 @@ public enum StorageVolumeType {
 
     public static String CREDENTIAL_MASK = "******";
 
+    private String dumpMaskedParams(Map<String, String> params) {
```

Comment:
possibly you can change `getProcNodeData()`, use this `dumpMaskedParams` for it to produce the ProcNodeData.